package.json

```diff
@@ -56,6 +56,7 @@
     "jest": "^27.5.1",
     "mini-css-extract-plugin": "^2.7.2",
     "new-github-issue-url": "^0.2.1",
+    "node": "^20.2.0",
```

Comment:
Why are node version changes necessary for this PR?  That seems like it doesn't belong here